openmmtools/utils/__init__.py

```diff
@@ -7,27 +7,6 @@
 This module provides a common API point for different utility functions that serve as convenience
 or help users in different miscellaneous tasks.
 """
-from .utils import (
-    _RESERVED_WORDS_PATTERNS,
-    deserialize,
-    find_all_subclasses,
-    get_available_platforms,
-    get_fastest_platform,
-    is_quantity_close,
-    platform_supports_precision,
-    quantity_from_string,
-    sanitize_expression,
-    math_eval,
-    RestorableOpenMMObject,
-    RestorableOpenMMObjectError,
-    serialize,
-    SubhookedABCMeta,
-    temporary_directory,
-    time_it,
-    Timer,
-    TrackedQuantity,
-    typename,
-    with_timer,
-)
+from .utils import *
```

Comment:
Do we really want to import everything into the namespace?

Reply:
Good question, there's no easy way to know what packages downstream are using, my first attempt was trying NOT to import everything, and only the things that openmmtools itself was needing, but it's hard to tell what other packages are using from `utils.py` so the easiest way is to import everything. I think in the future we probably want to remove the `utils.py` file and just have submodules that make more sense (such as what we did with the gentle equilibration utility function)

Reply:
> I haven't skimmed through to make sure that we don't have collisions, if this is exactly how it was working, then that is fine.

Yeah, it should be analogous to what was happening before the changes, such that users can still do `import openmmtools.utils` (or similar)

Reply:
Okay then I think it is okay! If we ever do a big breaking change, we can cleanup the API but for now lets make downstream life easy